src/Subscriptions/SubscriptionRegistry.php

```diff
@@ -53,12 +52,18 @@ class SubscriptionRegistry
      */
     protected $subscriptions = [];
 
+    /**
+     * @var \GraphQL\Type\Definition\ObjectType|null
+     */
+    protected $subscriptionType;
+
     public function __construct(ContextSerializer $serializer, StoresSubscriptions $storage, SchemaBuilder $schemaBuilder, ConfigRepository $configRepository)
     {
         $this->serializer = $serializer;
         $this->storage = $storage;
         $this->schemaBuilder = $schemaBuilder;
         $this->configRepository = $configRepository;
+        $this->subscriptionType = $schemaBuilder->schema()->getSubscriptionType();
```

Comment:
I think this might force building a schema on requests that do not use GraphQL at all.

Reply:
Agreed, lemme solve this another way!